Kickstarter-iOS/DataSources/PaymentMethodsDataSource.swift

```diff
@@ -0,0 +1,35 @@
+import KsApi
+import Library
+import UIKit
+
+final class PaymentMethodsDataSource: ValueCellDataSource {
+
+  public var deletionHandler: ((GraphUserCreditCard.CreditCard) -> Void)?
+
+  func load(creditCards: [GraphUserCreditCard.CreditCard]) {
+
+    self.set(values: creditCards,
+             cellClass: CreditCardCell.self,
+             inSection: 0)
+  }
+
+  override func configureCell(tableCell cell: UITableViewCell, withValue value: Any) {
+    switch (cell, value) {
+    case let (cell as CreditCardCell, value as GraphUserCreditCard.CreditCard):
+      cell.configureWith(value: value)
+    default:
+      assertionFailure("Unrecognized (cell, viewModel) combo.")
```

Comment:
We sure we want to crash if something goes wrong?

Reply:
Can someone double check that we're not currently testing this behaviour?

Reply:
This is in all of our data sources, there isn't really an alternative 🤷‍♂️ . Will `assertionFailure` crash in production though? Thought asserts are ignored if not running in debug.